node/common/core/common.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
   ConnectionString: require('./lib/connection_string.js'),
   endpoint: require('./lib/endpoint.js'),
   errors: require('./lib/errors.js'),
+  success: require('./lib/success.js'),
```

Comment:
Do we really want separate namespaces for success and error objects?  Makes sense that you wouldn't want to put MessageAccepted under the errors namespace, but on the other hand no one will need to use just the success objects or just the error objects--they need both.  I think we should put them together.